components/core/src/clp/ir/Constant.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef CLP_IR_CONSTANTS_HPP
+#define CLP_IR_CONSTANTS_HPP
+
+namespace clp::ir {
+constexpr char cIrExtension[] = "clp.zst";
```

Comment:
I think I wasn't clear in the previous comment (I commented in where this constant was used). I think it makes more sense to make it `.clp.zst`, as `.` should be a part of the extension. Say you want to check if a file is zstd compressed IR, you check if it ends with `.clp.zst` instead of `clp.zst` right? You could have `IamUsingclp.zst` which should not be considered as a clp file, neither with a clp IR extension

Reply:
Fair. I wasn't sure about if I need to add ".". After your explanation, adding "." to the extension makes more sense